src/uct/sm/self/self.c

```diff
@@ -196,7 +205,7 @@ static UCS_CLASS_INIT_FUNC(uct_self_iface_t, uct_md_h md, uct_worker_h worker,
 
     UCS_CLASS_CALL_SUPER_INIT(
             uct_base_iface_t, &uct_self_iface_ops,
-            &uct_base_iface_internal_ops, md, worker, params,
+            &uct_self_iface_internal_ops, md, worker, params,
```

Comment:
why needed? maybe do the same in `uct_base_iface_internal_ops`?

Reply:
Remove it. Thanks